test/fixtures/test-runner/no-isolation/global-hooks.js

```diff
@@ -0,0 +1,6 @@
+import('node:test').then((test) => {
+  test.before(() => console.log('before(): global'));
+  test.beforeEach(() => console.log('beforeEach(): global'));
+  test.after(() => console.log('after(): global'));
+  test.afterEach(() => console.log('afterEach(): global'));
+});
```

Comment:
```suggestion
});

```

Reply:
This change causes lint to fail.

Reply:
Done! Apologies for my mistake earlier, it's fixed now.